ui/src/dms/MultiDMInfoForm.tsx

```diff
@@ -87,11 +91,14 @@ export default function MultiDMInfoForm({ setOpen }: MultiDMInfoFormProps) {
           </div>
         </div>
         <footer className="flex items-center space-x-2">
-          <DialogPrimitive.Close asChild>
-            <button className="button ml-auto">Cancel</button>
-          </DialogPrimitive.Close>
+          <button
+            onClick={() => setEditing(false)}
+            className="secondary-button ml-auto"
+          >
+            Cancel
+          </button>
           <button type="submit" className="button">
-            Done
+            Save
```

Comment:
this should probably use the new loading state tracker thing and I don't think it's included in the other issues.

Reply:
Want to handle it in the loader mega-issue, or here?

Reply:
lets do here since it's not tracked in the mega